bay/images/awx-ee/context/Dockerfile

```diff
@@ -1,21 +1,27 @@
-ARG ANSIBLE_RUNNER_IMAGE=quay.io/ansible/ansible-runner:devel
-ARG PYTHON_BUILDER_IMAGE=quay.io/ansible/python-builder:latest
-
-FROM $ANSIBLE_RUNNER_IMAGE as galaxy
+ARG EE_BASE_IMAGE=quay.io/ansible/ansible-runner:latest
+ARG EE_BUILDER_IMAGE=quay.io/ansible/ansible-builder:latest
 
+FROM $EE_BASE_IMAGE as galaxy
 ARG ANSIBLE_GALAXY_CLI_COLLECTION_OPTS=
-ADD _build /build
+USER root
 
+ADD _build /build
 WORKDIR /build
+
 RUN ansible-galaxy role install -r requirements.yml --roles-path /usr/share/ansible/roles
 RUN ansible-galaxy collection install $ANSIBLE_GALAXY_CLI_COLLECTION_OPTS -r requirements.yml --collections-path /usr/share/ansible/collections
 
-FROM $PYTHON_BUILDER_IMAGE as builder
-ADD _build/requirements_combined.txt /tmp/src/requirements.txt
-ADD _build/bindep_combined.txt /tmp/src/bindep.txt
+FROM $EE_BUILDER_IMAGE as builder
+
+COPY --from=galaxy /usr/share/ansible /usr/share/ansible
+
+ADD _build/requirements.txt requirements.txt
+ADD _build/bindep.txt bindep.txt
+RUN ansible-builder introspect --sanitize --user-pip=requirements.txt --user-bindep=bindep.txt --write-bindep=/tmp/src/bindep.txt --write-pip=/tmp/src/requirements.txt
 RUN assemble
 
-FROM $ANSIBLE_RUNNER_IMAGE
+FROM $EE_BASE_IMAGE
+USER root
```

Comment:
Do we want the container to run as `root`? Or should we reset this to the user that the image ships with?

Reply:
This was one of the changes I hadn't expected when building this Dockerfile. Question for @steveworley

Reply:
This might be an upstream change with `ansible-builder` given that this docker file is generated and we haven't added anything to the builder files.

Reply:
You're right @steveworley https://github.com/ansible/ansible-builder/commit/82b4a40976a237e140a86a49cf848cd7b981b5b1#diff-d966ced5e7d9ab32dc150a97efca27b8b3d89fe939d8aa7001afafcd0f620e36L434. It was swapped out to deal with permissions errors during build stage.
Effectively there's no difference introduced in this branch. I've run a build of it and the 4.x branch and they're both set to `root`.